ivre/web/utils.py

```diff
@@ -472,26 +472,32 @@ def add_unused(neg, param, value):
             flt = db.view.flt_and(flt, db.view.searchxp445())
         elif param == "ssl-ja3-client":
             flt = db.view.flt_and(flt, db.view.searchja3client(
-                value_or_hash=value,
+                value_or_hash=(None if value is None else
+                               utils.str2regexp(value)),
                 neg=neg
             ))
         elif param == "ssl-ja3-server":
             if value is None:
                 # There are no additional arguments
                 flt = db.view.flt_and(flt, db.view.searchja3server(neg=neg))
             else:
-                splitted = value.split(':', 1)
-                if len(splitted) == 1:
+                split = [v if v else None for v in value.split(':', 1)]
```

Comment:
Why don't you apply `utils.str2regexp` here instead of line 490/498/500 ? Like : 
``` 
split = [utils.str2regexp(v) if v else None for v in value.split(':', 1)]
``` 
If so, you could remove the if conditions line 489, 497 and 499 and give directly variables `srv` and `cli` .